Sources/CocoaLumberjack/DDLog.m

```diff
@@ -1037,10 +1037,22 @@ - (instancetype)initWithMessage:(NSString *)message
     return self;
 }
 
+static __inline__ __attribute__((__always_inline__)) BOOL _nullable_strings_equal(NSString* _Nullable lhs, NSString* _Nullable rhs)
```

Comment:
Foundation provides an NS_INLINE macro for `__inline__ __attribute__((__always_inline__))`